docs/develop/external-calls.md

```diff
@@ -1,28 +1,29 @@
 ---
 title: 'External API call support in Office Scripts'
 description: 'Support and guidance for making external API calls in an Office Script.'
-ms.date: 09/24/2020
+ms.date: 12/28/2020
 localization_priority: Normal
 ---
 
 # External API call support in Office Scripts
 
-The Office Scripts platform doesn't support calls to [external APIs](https://developer.mozilla.org/docs/Web/API). However, these calls can be run under the right circumstances. External calls can be only be made through the Excel client, not through Power Automate [under normal circumstances](#external-calls-from-power-automate).
+Script authors shouldn't expect consistent behavior when using [external APIs](https://developer.mozilla.org/docs/Web/API) during the platform's preview phase. This is due how the JavaScript runtime manages interacting with the workbook. The script may end before the API call completes (or its `Promise` is fully resolved). As such, do not rely on external APIs for critical script scenarios.
 
-Script authors shouldn't expect consistent behavior when using external APIs during the platform's preview phase. This is due how the JavaScript runtime manages interacting with the workbook. The script may end before the API call completes (or its `Promise` is fully resolved). As such, do not rely on external APIs for critical script scenarios.
+Calls to external APIs can be only be made through the Excel client, not through Power Automate [under normal circumstances](#external-calls-from-power-automate).
```

Comment:
Iirc you can make fetch calls in the PA AzFx runner, though there were some details there around admin lockdown. Please check with Yutao or Danny.